### PR TITLE
Update cv / view cv

### PR DESCRIPTION
**Pull request by Jehan98**

### CODE REVIEW

File: components/student_dashboard/UserDetails.js [hunk 1]
```diff
@@ -8,14 +8,87 @@ import {
...
+				const response = await setCvUrl(user.uuid, cvURL);
+				
+				if(response && response.status === 200){
+					toast({
+						title: "Uploaded Successfully",
+						description: "Your CV uploaded successfully",
+						status: "success",
+						duration: 5000,
+						isClosable: true,
+					})
```
[the remark this answers is not in the document]
Comment (chandralegend) [reply components/student_dashboard/UserDetails.js:57]: @Jehan98 use Prettier or any other formatting tool in your code editor. to keep a consistent structure

File: components/student_dashboard/UserDetails.js [hunk 2]
```diff
@@ -8,14 +8,87 @@ import {
 	Button,
 	useDisclosure,
 	Center,
+	Input,
+	useToast
 } from "@chakra-ui/react";
 import { useAuth } from "../../lib/auth";
```
Comment (chandralegend) [inline components/student_dashboard/UserDetails.js:14]: seperate our imports from module imports

Comment (chandralegend) [inline components/student_dashboard/UserDetails.js:14]: @Jehan98 Seperate Module Imports from our imports

File: components/student_dashboard/UserDetails.js [hunk 3]
```diff
@@ -8,14 +8,87 @@ import {
...
 const UserDetails = () => {
 	const cardBackground = useColorModeValue("gray.100", "gray.900");
 	const { isOpen, onOpen, onClose } = useDisclosure();
 	const { user } = useAuth();
+	const toast = useToast()
+	const router = useRouter()
+	const hiddenFileInput = React.useRef(null);
+
+	const [uploadingCV, setUploadingCV] = useState(false);
+	const [cvURL, setUserCvURL] = useState("");
```
[the remark this answers is not in the document]
Comment (chandralegend) [reply components/student_dashboard/UserDetails.js:31]: @Jehan98 use a tool such as Prettier on your code editor to maintain a consistence in your code

File: lib/api.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import axios from "axios";
 
 const api = axios.create({
 	baseURL:
-		"https://us-central1-careerfair-development.cloudfunctions.net/api/v1",
```
Comment (chandralegend) [inline lib/api.js:5]: keep the production baseUrl

**Review by chandralegend — CHANGES_REQUESTED** [review]
@Jehan98 Read the Reviews and Correct them.

**Review by Jehan98 — COMMENTED** [review]
Now?

**Review by chandralegend — APPROVED** [review]
Good Work

**Pull request merged**